src/DeviceManager.Api/appsettings.json

```diff
@@ -5,7 +5,7 @@
     "DatabaseType": "MsSql",
 
     // Sql Database
-    "DefaultConnection": "Server=(localdb)\\mssqllocaldb;Database=DeviceDb;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=192.168.96.1,1433;Database=DeviceDb;User Id=sa;Password=Abc@1234;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
Please don't merge this line. I mistakenly checked in this line.

Reply:
Hey @rudreshgp Did you mean not use this one: `"DefaultConnection": "Server=192.168.96.1,1433;Database=DeviceDb;User Id=sa;Password=Abc@1234;Trusted_Connection=True;MultipleActiveResultSets=true"`

Reply:
Yes. 
Also, I was on holiday from the last month. I will try to contribute from now on.

Reply:
Hey @rudreshgp Ah ok. Nice to hear that.